fastlane/Snapfile

```diff
@@ -1,17 +1,16 @@
 devices([
-  "iPhone 8",
-  "iPhone 8 Plus",
-  "iPhone 11",
-  "iPhone 11 Pro",
-  "iPhone 11 Pro Max",
-  "iPad Pro (12.9-inch) (3rd generation)"
+  # "iPhone SE (3rd generation)",
```

Comment:
is this temporary? seems like we should still snapshot a variety of devices

Reply:
Yes, I'll re-enable the relevant devices to get the needed screen sizes before merge

Reply:
done